File: server/src/WebAPI.Contracts/User/LoginUserRequest.cs
namespace WebAPI.Contracts.User;

public sealed record LoginUserRequest(
    string Email,
    string Password)
{
    /// <example>admin@example.ru</example>
    public string Email { get; init; } = Email;
    /// <example>admin</example>
    public string Password { get; init; } = Password;
}

File: new-client/src/shared/model/LoginUserRequest.ts
import { z } from "zod"

export const LoginUserRequest = z.object({
  email: z.string().min(3, { message: `Строка должна содержать минимум 3 символа` }).max(50),
  password: z.string().min(3, { message: `Строка должна содержать минимум 3 символа` }).max(50)
})
